crates/bevy_asset/src/lib.rs

```diff
@@ -1,3 +1,143 @@
+//! In the context of game development, an "asset" is a piece of content that is loaded from disk and displayed in the game.
+//! Typically, these are authored by artists and designers (in contrast to code),
+//! are relatively large in size, and include everything from textures and models to sounds and music to levels and scripts.
+//!
+//! This presents two main challenges:
+//! - Assets take up a lot of memory; simply storing a copy for each instance of an asset in the game would be prohibitively expensive.
+//! - Loading assets from disk is slow, and can cause long load times and delays.
+//!
+//! These problems play into each other, for if assets are expensive to store in memory,
+//! then larger game worlds will need to load them from disk as needed, ideally without a loading screen!
```

Comment:
```suggestion
//! then larger game worlds will need to load them from disk as needed, ideally without a loading screen.
```

Small nit, emphasis doesn't seem necessary.